src/util/__tests__/markdownToText.ts

```diff
@@ -0,0 +1,465 @@
+    - g
+`)
+  })
+
+  it('should handle headings in lists', () => {
+    const markdown = `
+- List Item 1
+- ## List Item 2
+- List Item 3
+`
+    expect(markdownToText(markdown)).toBe(`
+- List Item 1
+- List Item 2
+- List Item 3
+`)
+  })
+
+  it('should import emphasis', () => {
+    const markdown = `
+*italic*
+_italic_
+**bold**
+__bold__
+***bold italic***
+~~strikethrough~~
+`
+    expect(markdownToText(markdown)).toBe(`
+- <i>italic</i> <i>italic</i> <b>bold</b> <b>bold</b> <i><b>bold italic</b></i> <strike>strikethrough</strike>
+`)
+  })
+
+  it('should import lists', () => {
+    const markdown = `
+1. First ordered item
+2. Second ordered item
+   * Unordered sub-list that collapses
+* Unordered item
+- Another unordered item
++ Yet another unordered item
+
+# The next list will collapse to this heading
+1. First ordered item
+2. Second ordered item
+`
+    expect(markdownToText(markdown)).toBe(`
+- ::
+  - =scope
+  - =ordered
+  - First ordered item
+  - Second ordered item
+    - Unordered sub-list that collapses
+- Unordered item
+- Another unordered item
+- Yet another unordered item
+- The next list will collapse to this heading
+  - =ordered
+  - First ordered item
+  - Second ordered item
+`)
+  })
+
+  it('should import links and resolve reference links', () => {
+    const markdown = `
+[Inline link](https://example.com)
+[Inline link with title](https://example.com "Example")
+
+[Reference link][ref]
+
+[ref]: https://example.com
+`
+    expect(markdownToText(markdown)).toBe(`
+- <a href="https://example.com">Inline link</a> <a href="https://example.com">Inline link with title</a>
+- <a href="https://example.com">Reference link</a>
+`)
+  })
+
+  it('should import images', () => {
+    const markdown = `
+![Alt text](https://example.com/image.jpg)
+
+![Alt text with title](https://example.com/image.jpg "Image title")
+`
+    expect(markdownToText(markdown)).toBe(`
+- =image
+  - https://example.com/image.jpg
+- =image
+  - https://example.com/image.jpg
+    - Image title
+`)
+  })
+
+  it('should import inline images as html tag', () => {
+    const markdown = `
+Here's an inline image ![Alt text](https://example.com/image.jpg "Image").
+`
+    expect(markdownToText(markdown)).toBe(`
+- Here&#39;s an inline image <img src="https://example.com/image.jpg" alt="Alt text" title="Image" />.
+`)
+  })
+
+  it('should import code', () => {
+    const markdown = `
+Inline \`code\` has \`back-ticks around\` it.
+
+\`\`\`javascript
+var s = "JavaScript";
+alert(s);
+\`\`\`
+`
+    expect(markdownToText(markdown)).toBe(`
+- Inline <code>code</code> has <code>back-ticks around</code> it.
+- var s = "JavaScript";&#10;alert(s);
+  - =code
+`)
+  })
+
+  it('should import a two-column table at the root', () => {
+    const markdown = `
+| Column 1 | Column 2 |
+|----------|:--------:|
+| Left     | Right    |
+| 1        | 2        |
+| apple    | orange   |
+`
+    expect(markdownToText(markdown)).toBe(`
+- ::
+  - =scope
```

Comment:
This shouldn't be scoped, as the table does not have any siblings.

Added tests in aa6b3838be.